app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return 'All friends should be vaccinated'
```

Comment:
The message 'All friends should be vaccinated' is generic and does not indicate which friend is not vaccinated. Add additional information, such as the index or name of the failed friend.

Reply:
Another thing, the code is not taking advantage of the custom messages defined in the classes you created and did not define a message in the errors.py file.